microsoft/PowerToys
# [PowerRename] Move PowerRename module installation into separate folder.

## Summary of the Pull Request
Move `PowerRename` output files into separate folder within modules folder to make build and installation folders more organized.


## PR Checklist
* [x] Applies to #3812


## Validation Steps Performed
1. Build and install PowerToys Release x64 version.
2. Check installation folder (`C:\Program Files\PowerToys\modules`) for `PowerRename` folder.
3. Test `PowerRename` functionalities to make sure it works as expected.
